test/flux_calculator_tests.jl

```diff
@@ -1,6 +1,7 @@
 import Test: @test, @testset, @test_throws
 import StaticArrays
 import ClimaCore as CC
+import ClimaParams
```

Comment:
is this needed?

Reply:
without it I get `MethodError: no method matching ThermodynamicsParameters(::Type{Float32})`, but I'm not sure why it's coming up in this PR

Reply:
Ahh actually, it's because this test is called after other tests that load ClimaParams in runtests.jl. I noticed that we need it here too when I was running each test individually